rainmaker-pgr/src/main/java/org/egov/pgr/contract/ServiceReqSearchCriteria.java

```diff
@@ -70,8 +70,8 @@
 	@JsonProperty("createdBy")
 	private String createdBy;
 	
-	@JsonProperty("noOfRecords")
-	private Long noOfRecords;
+	@JsonProperty("limit")
```

Comment:
Is this change required? Now that we are not using searcher

Reply:
Not needed anymore.